tests/unittest/bindings/test_executor_bindings.py

```
@@ -1402,12 +1403,13 @@ def test_scheduler_config():
     dynamic_batch_config = trtllm.DynamicBatchConfig(True, True, 128)
     config = trtllm.SchedulerConfig(capacity_scheduler_policy,
                                     context_chunking_policy,
-                                    dynamic_batch_config)
+                                    dynamic_batch_config, False)
     assert config.capacity_scheduler_policy == capacity_scheduler_policy
     assert config.context_chunking_policy == context_chunking_policy
     assert config.dynamic_batch_config.enable_batch_size_tuning == True
```

Ruff (0.15.18)

[error] Avoid equality comparisons to `True`; use `...:` for truth checks

Replace comparison

(E712)